metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_schema_gen.py

```diff
@@ -653,14 +654,11 @@ def _process_view(
             self.report.report_dropped(table_identifier.raw_table_name())
             return
 
-        if self.store_table_refs:
-            table_ref = str(
-                BigQueryTableRef(table_identifier).get_sanitized_table_ref()
-            )
-            self.table_refs.add(table_ref)
-            if self.config.lineage_parse_view_ddl and view.view_definition:
-                self.view_refs_by_project[project_id].add(table_ref)
-                self.view_definitions[table_ref] = view.view_definition
+        table_ref = str(BigQueryTableRef(table_identifier).get_sanitized_table_ref())
+        self.view_snapshot_refs.add(table_ref)
```

Comment:
What was wrong with earlier logic of keeping all table_refs in single collection ?

Reply:
I'd prefer the earlier one if it behaves same.

Reply:
it was completely depend on table_lineage/usage_statistics and use_queries_v2 so if all this flags are false 
then we wont be able to see the view lineage
Thats the reason i have removed